package.json

```diff
@@ -116,6 +117,7 @@
     "flow-bin": "~0.117.0",
     "flow-typed": "^2.5.1",
     "html-loader": "^0.5.5",
+    "html-to-text": "^5.1.1",
```

Comment:
Why did you switch to `htmlparser2`? Probably you should remove `html-to-text` then.

Reply:
Oh. Forgot to remove it. `html-to-text` had a large overhead. @Markl recommended htmlparser2. It is also a lot faster than before.